recipe/meta.yaml

```
@@ -7,9 +7,12 @@ package:
 source:
   url: https://pypi.io/packages/source/C/Cython/Cython-{{ version }}.tar.gz
   sha256: 6a0d31452f0245daacb14c979c77e093eb1a546c760816b5eed0047686baad8e
+  patches:
+    # part of next release
```

Would suggest including a reference to the PR in the comment so we can track when this can be dropped

```suggestion
    # xref: https://github.com/cython/cython/pull/4236
```